BUILDING.md

```diff
@@ -172,14 +172,13 @@ Binaries at <https://nodejs.org/download/release/> are produced on:
 | darwin-x64 (and .pkg) | macOS 10.15, Xcode Command Line Tools 11 with -mmacosx-version-min=10.13 |
 | linux-arm64           | CentOS 7 with devtoolset-6 / GCC 6                                       |
 | linux-armv7l          | Cross-compiled on Ubuntu 16.04 x64 with [custom GCC toolchain](https://github.com/rvagg/rpi-newer-crosstools)   |
-| linux-ppc64le         | CentOS 7 with devtoolset-6 / GCC 6 <sup>[7](#fn7)</sup>                  |
-| linux-s390x           | RHEL 7 with devtoolset-6 / GCC 6 <sup>[7](#fn7)</sup>                    |
-| linux-x64             | CentOS 7 with devtoolset-6 / GCC 6 <sup>[7](#fn7)</sup>                  |
-| sunos-x64             | SmartOS 18 with GCC 7                                                    |
+| linux-ppc64le         | CentOS 7 with devtoolset-6 / GCC 8 <sup>[8](#fn8)</sup>                  |
+| linux-s390x           | RHEL 7 with devtoolset-6 / GCC 8 <sup>[8](#fn8)</sup>                    |
+| linux-x64             | CentOS 7 with devtoolset-6 / GCC 8 <sup>[8](#fn8)</sup>                  |
```

Comment:
We actually build with devtoolset-8/gcc-8 at this time. devtoolset-6 is gcc-6 (the "6" is the underlying gcc version). Is this a minimum, or a "what we use"? Current compiler levels from daily master:

- linux-ppcle: gcc8.3.1: https://ci.nodejs.org/job/node-test-commit-plinux/32233/nodes=centos7-ppcle/consoleFull
- linux-s390x: gcc8.3.1: https://ci.nodejs.org/job/node-test-commit-linuxone/20275/nodes=rhel7-s390x/consoleFull
- linux-x64: gcc8.2.1: https://ci.nodejs.org/job/node-test-commit-linux/34237/nodes=centos7-64-gcc8/consoleFull

Reply:
Good catch — will fix in the morning. This is what we use. The section above this one is the minimum.

Reply:
Updated references to devtoolset-6 to devtoolset-8 (including Linux arm64).